app/models/serial_work.rb

```diff
@@ -26,9 +26,9 @@ def delete_empty_series
 
   # Ensure series bookmarks are reindexed when a new work is added to a series
   def update_series_index
+    series.enqueue_to_index
     if $rollout.active?(:start_new_indexing)
-      series.enqueue_to_index
-      series.bookmarks.each(&:enqueue_to_index)
+      IndexQueue.enqueue_ids(Bookmark, series.bookmarks.pluck(:id), :main)
```

Comment:
IndexQueue will index for both versions of ES (if we have `start_new_indexing` but not `stop_old_indexing`), so `update_series_index` will end up indexing the bookmarks 4 times: once per ES version, and once per queue (main and background). I don't think we can do background for 0.90, main for 6 here.

If we want to be extra careful, we could use background, and add an "upgrade transition" comment for switching to main later.

Reply:
Hmm. That would be a problem with the code that's currently on staging, too, wouldn't it?

Reply:
It is. :<

Reply:
I'm afraid to ask if there's similar code in other places as well.

Reply:
This is the only place I can find that uses `stop_old_indexing` like this.